geo_activity_playground/core/activity_parsers.py

```diff
@@ -42,13 +44,13 @@ def read_activity(path: pathlib.Path) -> pd.DataFrame:
         try:
             df = read_tcx_activity(path, opener)
         except xml.etree.ElementTree.ParseError as e:
-            raise ActivityParseError(f"Syntax error in TCX file {path=}") from e
+            raise ActivityParseError("Syntax error in TCX file") from e
     elif file_type in [".kml", ".kmz"]:
         df = read_kml_activity(path, opener)
     elif file_type == ".csv":  # Simra csv export
         df = read_simra_activity(path)
     else:
-        raise ActivityParseError(f"Unsupported file format: {file_type}")
+        raise ActivityParseError(f"Unsupported file format: {file_type} of file: {path}")
```

Comment:
The path will be added in the `import_from_directory` function, so we don't need to repeat it here.